muxplore/main.c

```diff
@@ -2293,7 +2273,7 @@ int main(int argc, char *argv[]) {
         lv_obj_clear_flag(ui_lblScreenMessage, LV_OBJ_FLAG_HIDDEN);
     }
 
```

Comment:
pthread_cancel was causing a segmentation fault when using dual card setup.  updated to track whether gen_item_thread was created successfully so that pthread_join and pthread_cancel do not get called when the thread does not exist

Reply:
Ah this resolves the issue I was finding with bcats change then.  Just waiting on the internal PR small fixes and we'll be ready to rock and roll!